proposed/0013-immutable-data-naming/0013-immutable-data-naming.md

```diff
@@ -13,37 +13,28 @@ This RFC outlines the system components and design for how the three immutable d
 
 ## Rationale
 
-The SAFE Network keeps multiple copies of one original data not only for the purpose of avalability but also for secure reason.  It has been defined three different types currently : Normal, Backup and Sacrificial.  They are currently having the same name (hash of the content), which means it will be the same group of DataManager to handle them.
+The SAFE Network keeps multiple copies of a single ImmutableData chunk, not only for availability but also for security reasons.  To that end, three different types of ImmutableData have been defined: Normal, Backup and Sacrificial.  These differ only in how their name is calculated; their contents are identical.
 
-Making three different types copy bearing different but determinic name will increase the difficultiy of any possible attack, as it will be three different groups, instead of just one group, need to be tackled at the same time.
+Having three different types bearing different but deterministic names will increase the difficulty of any attack, as it will be three different groups of Vault, which need to be tackled at the same time.
 
 ## Supported Use-Cases
 
-For client perspective, the use-case keeps the same : storing or fetching immutable data.  Though client may need to be aware of the existing of different types and calculations of name. 
+From the client perspective, the use-case stays the same: storing or fetching immutable data.  However, the client may need to be aware of the existence of different types.
 
 ## Expected Outcome
 
-The different immutable data type copy bearing different name, which will be handled by different DataManager group.
+Replication of the different immutable data types bearing different names, which will be handled by different DataManager groups.
```

Comment:
Is it worth re-iterating here the Data will go the the correct `ǸaeManager` for the data. So same as all other data. The Vault will then manage each type as per normal. It requires to do no calculation on whether it should have it etc. 

The difference will be in handling it from the main copy DM who may request backup or sacrificial copies. Then again Backup may require main or sacrificial copies.  etc. 

Sacrificial will not try and get copies unless there is space in the group and it has lost both copies.